templates/test/ci/prow-private/kustomization.yaml

```diff
@@ -4,6 +4,7 @@ namespace: default
 resources:
   - ../../../flavors/private
   - cni-resource-set.yaml
+  - ../../../cluster-api-helm/calico.yaml
 patchesStrategicMerge:
   - ../patches/tags.yaml
   - ../patches/controller-manager.yaml
```

Comment:
you should be able to remove the `patches/cluster-cni.yaml` patch in the private cluster test and remove the TODO comment about CNI being installed via ClusterResourceSet in the privatecluster test spec